README.md

```diff
@@ -1 +1,112 @@
-TODO
+# WhyLabs Toolkit
+
+The WhyLabs Toolkit package contains helper methods to help users interact with our internal APIs. Users will benefit from using it if they want to abstract some of WhyLabs' internal logic and also automate recurring API calls.
+
+## Configuration
+In order to configure `whylabs_toolkit`, you will need to set `WHYLABS_API_KEY` as an environment variable. With that, the package will be able to authenticate with WhyLabs' API endpoints. You can configure a token for your account directly on the platform. 
+
+To manage dependencies we use [Poetry](https://python-poetry.org/) and also a handful of `Makefile` commands. To install all necessary dependencies and activate the virtual environment, run:
+
+```bash
+make setup && poetry shell
+```
+
+## Usage
+In here we will describe some examples on how to use the package. You can also check [the integration tests directory](./tests/) to have more insights on how things are built to be used.
+### Models
+Users can change their model type between `REGRESSION`, `CLASSIFICATION` and `EMBEDDINGS`, using the models helpers, as the example shows:
+```python
+from whylabs_toolkit.helpers.models import update_model_metadata
```

Comment:
Why do we make imports so nested? Can it be something like:

```
from whylabs_toolkit import datasets
```

And then you can have:

```
dataset.create(...)
```
And
```
daaset.update(...)
```

Reply:
yes, definitely. i will defer this to the next PR, which is dedicated to refactor the codebase pattern + making imports look nicer.